### PR TITLE
WSL 2 backend updates for Edge release

### PR DESCRIPTION
**Pull request by usha-mandya**
WSL 2 backend updates for Docker Desktop Edge 2.2.3.0 release.

Changes include:
- Modified the prereqs and installation instructions
- Updated screenshots to reflect the latest UI
- Added new sections on developing with Docker and WSL 2, best practices
- Moved Feedback to the end of the page
- Updated headings to h2 to match the rest of the docs

### CODE REVIEW

**Comment by usha-mandya** [comment]
@simonferquel @nebuk89 Could you PTAL?

**Comment by usha-mandya** [comment]
You can preview the updated page here - https://angry-visvesvaraya-c62c8e.netlify.com/docker-for-windows/wsl-tech-preview/.

File: docker-for-windows/wsl-tech-preview.md [hunk 1]
```diff
@@ -1,55 +1,92 @@
...
+8. Click **Apply & Restart**.
+
+## Develop with Docker and WSL 2
+
+The following section describes how to start developing your applications using Docker and WSL 2. We recommend that you have your code in your default Linux distribution for the best development experience using Docker and WSL 2. After you have enabled WSL 2 on Docker Desktop, you can start working with your code inside the Linux distro and ideally with your IDE still in Windows. This workflow can be pretty straightforward if you are using [VSCode](https://code.visualstudio.com/download).
+
+1. Open VSCode and install the [Remote - WSL](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl) extension. This extension allows you to work with a remote server in the Linux distro and your IDE client still on Windows.
+2. Now, you can start working in VSCode remotely. To do this, open your terminal and type:
+
+    `Wsl`
```
Comment (stephen-turner) [inline docker-for-windows/wsl-tech-preview.md]: I don't have a WSL-capable computer I can test this on, but does this really have an upper case `W`?

File: docker-for-windows/wsl-tech-preview.md [hunk 2]
```diff
@@ -1,55 +1,92 @@
...
-    `wsl -l -v`
+    Alternatively, you can type the name of your default Linux distro in your Start menu, open it, and then run `code` .
+3. When you are in VSCode, you can use the terminal in VSCode to pull your code and start working natively from your Windows machine.
 
-    To upgrade to v2, run:
+## Best practices
 
-    `wsl --set-version <distro name> 2`
-7. When Docker Desktop restarts, go to **Settings** > **Resources** > **WSL Integration** and then select from which WSL 2 distributions you would like to access Docker.
+- To get the best out of the file system performance, we recommend that you avoid mounting from the Windows file system (even on a WSL distro). For example, avoid `docker run -v /mnt/c/users:/users`.
```
Comment (stephen-turner) [inline docker-for-windows/wsl-tech-preview.md]: I want this to be followed by "Instead, mount [something else]".

File: docker-for-windows/wsl-tech-preview.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/wsl-tech-preview.md]: @simonferquel Could you confirm this? I got this command from the blog - https://www.docker.com/blog/docker-desktop-for-windows-home-is-here/

File: docker-for-windows/wsl-tech-preview.md (hunk 2 above)
Comment (usha-mandya) [reply docker-for-windows/wsl-tech-preview.md]: @simonferquel Could you provide some guidance here? Thanks.

File: docker-for-windows/wsl-tech-preview.md (hunk 1 above)
Comment (simonferquel) [reply docker-for-windows/wsl-tech-preview.md:74]: On Windows, executables are case-insensitive. But `wsl.exe`  feels better than `Wsl.exe`.

File: docker-for-windows/wsl-tech-preview.md (hunk 2 above)
Comment (simonferquel) [reply docker-for-windows/wsl-tech-preview.md:85]: "Instead prefer mounting files living in your distro's local filesystem such as in: `docker run -v ~/my-project:/sources <my-image>`"

File: docker-for-windows/wsl-tech-preview.md (hunk 1 above)
Comment (usha-mandya) [reply docker-for-windows/wsl-tech-preview.md:74]: Great, thanks for confirming.

**Pull request merged**